scripts/update-release-list.py

```diff
@@ -78,28 +78,25 @@ def write_new_rels(new_rels, latest_rel):
             lines += [l]
 
     print('New releases: ', end='')
-    f = open(latest_file, 'w')
-    for i, rel in enumerate(new_rels):
-        y = rel['published_at'][:4]
-        m = rel['published_at'][5:7]
-        if y != last_y:
-            f.writelines(lines)
-            f.close()
-            lines = []
-            last_y = y
-            f = open(f'Releases-in-{y}.md', 'w')
```

Comment:
The reason why I didn't use `with open() as f` is that I needed to reassign `f` here, and the reassigned `f` should be used in the rest of the `for` loop.

I haven't tried to run this PR, but it looks that the behavior is changed.